include/revng/Pipes/AllFunctions.h

```diff
@@ -11,7 +11,7 @@
 namespace revng::pipes {
 
 pipeline::TargetsList
-compactFunctionTargets(const model::Binary &Model,
+compactFunctionTargets(const TupleTree<model::Binary> &Model,
```

Comment:
It seems to me that you squashed changes that could have been a standalone commit in this one, please try to keep them isolated next time.

Reply:
yeah, the patch set got convoluted due to rebasing on pervasive changes and at one point it was easier to just sqush